lxqt-base/lxqt-meta/lxqt-meta-0.13.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit versionator
+
+DESCRIPTION="Meta ebuild for LXQt, the Lightweight Desktop Environment"
+HOMEPAGE="http://lxqt.org/"
```

Comment:
switch homepage to https again everywhere

Reply:
@dracwyrm, also one more nitpick, as per upstream site the code is being destributed under `GNU LGPL 2.1+` __and__  `GNU GPL 2.0+` while I see you included only `GNU LGPL 2.1+`